lib/generator/code_generator.dart

```diff
@@ -32,37 +32,35 @@ class GraphQLCodeGenerators {
     return mutationObjectType;
   }
 
-  generateTypes(
+  String generateTypes(
       String namespace,
       Map<String, String> fragments,
       Map<String, DartType> types,
       List<ObjectType> responseTypes,
       ObjectType mutationObjectType) {
     var result = '';
     var emitter = DartEmitter(Allocator());
-    Map<String, Class> classes = {};
+    var classes = <Class>[];
 
-    List<String> enumString =
+    var enumString =
         EnumGenerator().enumGenerator(responseTypes, namespace: namespace);
-    Map<String, Class> interfaces = InterfaceGenerator()
+    var interfaces = InterfaceGenerator()
         .generate(types, responseTypes, namespace: namespace);
-    Map<String, Class> unions =
-        UnionGenerator().generate(responseTypes, namespace: namespace);
-    Map<String, Class> inputs = InputObjectGenerator()
+    var unions = UnionGenerator().generate(responseTypes, namespace: namespace);
+    var inputs = InputObjectGenerator()
         .generate(types, responseTypes, namespace: namespace);
-    Map<String, Class> objects = ObjectClassGenerator()
+    var objects = ObjectClassGenerator()
         .generate(types, responseTypes, namespace: namespace);
 
     classes.addAll(interfaces);
```

Comment:
Please use the spread operator here instead of `addAll`:
https://medium.com/flutter-community/whats-new-in-dart-2-3-1a7050e2408d